### PR TITLE
fix incorrect spell

### PR DESCRIPTION
**Pull request by gosoon**

### CODE REVIEW

File: keps/sig-node/20191118-cgroups-v2.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ A proposal to add support for cgroups v2 to kubernetes.
 
 The new kernel cgroups v2 API was declared stable more than two years
 ago. Newer features in the kernel such as PSI depend upon cgroups
-v2. groups v1 will eventually become obsolete in favor of cgroups v2.
+v2. cgroups v1 will eventually become obsolete in favor of cgroups v2.
```
Comment (yuzhiquan) [inline keps/sig-node/20191118-cgroups-v2.md:55]: May be `c` should be capitalized...

Comment (gosoon) [reply keps/sig-node/20191118-cgroups-v2.md:55]: You are right. I will submit a new pr.

**Comment by gosoon** [comment]
I will submit a new pr.

**Pull request closed without merging**